examples/bull-spread/bull-spread.go

```diff
@@ -0,0 +1,161 @@
+	}
+
+	service := Service{tdClient: tdClient, mdClient: mdClient, acct: acct}
+	return &service, nil
+}
+
+func main() {
+	svc, err := NewService()
+	if err != nil {
+		log.Fatalf("failed to intilize service: %v", err)
+	}
+
+	underlying := "INTC"
+	td, err := svc.mdClient.GetLatestTrade(underlying, marketdata.GetLatestTradeRequest{Feed: "iex"})
+	if err != nil {
+		log.Fatalf("getting latest trade for symbol %s: %v", underlying, err)
+	}
+	px := td.Price
+
+	// list contracts that are expiring at least a week from now and strike is GTE latest trade
+	now := time.Now()
+	dt := civil.DateOf(time.Date(now.Year(), now.Month(), now.Day()+oneWeek, 0, 0, 0, 0, time.UTC))
+
+	// Select two contracts for the two legs of the spread:
+	// 1. long leg, strike A at latest trade
+	var (
+		long  alpaca.OptionContract
+		short alpaca.OptionContract
+	)
+	{
+		req := alpaca.GetOptionContractsRequest{
+			UnderlyingSymbols: underlying,
+			Status:            alpaca.OptionStatusActive,
+			ExpirationDateGTE: dt,
+			StrikePriceGTE:    decimal.NewFromFloat(px), // strike A
+			TotalLimit:        1,
+		}
+		contracts, err := svc.tdClient.GetOptionContracts(req)
+		if err != nil {
+			log.Fatalf("listing contracts: %v", err)
+		}
+		if len(contracts) < 1 {
+			log.Fatal("no contracts to choose from for the long leg")
+		}
+		long = contracts[0]
+		log.Printf("-> contract to be bought long: %s", long.Symbol)
+	}
+	// 2. short leg, strike B at $10 above latest trade
+	{
+		req := alpaca.GetOptionContractsRequest{
+			UnderlyingSymbols: underlying,
+			Status:            alpaca.OptionStatusActive,
+			ExpirationDateGTE: dt,
+			StrikePriceGTE:    decimal.NewFromFloat(px + 10), // strike B
+			TotalLimit:        1,
+		}
+		contracts, err := svc.tdClient.GetOptionContracts(req)
+		if err != nil {
+			log.Fatalf("listing contracts: %v", err)
+		}
+		if len(contracts) < 1 {
+			log.Fatal("no contracts to choose from for the short leg")
+		}
+		short = contracts[0]
+		log.Printf("-> contract to be shorted: %s", short.Symbol)
+	}
+
+	// Bullish Call Spread: You profit if the stock price is between the strike (A) and the
+	// higher strike (B). The maximum profit occurs when the stock is at or above strike B, but
+	// the profit is capped at the difference between the two strikes minus the premium paid (cost of the spread).
+	qty := decimal.NewFromInt(2) // instances of the strategy to be placed
+	order, err := svc.tdClient.PlaceOrder(alpaca.PlaceOrderRequest{
+		Qty:         &qty,
+		TimeInForce: alpaca.Day,
+		Type:        alpaca.Market,
+		OrderClass:  alpaca.MLeg,
+		Legs: []alpaca.Leg{
+			{
+				Symbol:         long.Symbol,
+				Side:           alpaca.Buy,
+				PositionIntent: alpaca.BuyToOpen,
+				RatioQty:       decimal.NewFromInt(1),
+			},
+			{
+				Symbol:         short.Symbol,
+				Side:           alpaca.Sell,
+				PositionIntent: alpaca.SellToOpen,
+				RatioQty:       decimal.NewFromInt(1),
+			},
+		},
+	})
+	if err != nil {
+		panic(err)
```

Comment:
[minor] Why not `log.Fatal`?

Reply:
I'm getting "submit mleg order is disabled in production (HTTP 403, Code 40310000)" on the production paper environment.

Reply:
> I'm getting "submit mleg order is disabled in production (HTTP 403, Code 40310000)" on the production paper environment.
Yes, we are launching on the 23rd of December.

Reply:
Fixed.
7f5ef5cad60686b69fb44fc685de6a282e972c21